### DISCUSSION TITLE
Jeedom plugin - How to help your jeedom to collect your pool data

### DISCUSSION THREAD

**Discussion by mguyard** [iopool/community#2] [Community development]
Hi all,

I work to develop a plugin for Jeedom (https://www.jeedom.com/).
Jeedom is a french domotic solution who works with a market of plugins.

iopool plugin will use API call to collect and store all pools informations to Jeedom. With that, you will be able to exploit data programmatically

It's a work in progress

**Comment by john-does-it** [comment]
<3

**Comment by mehdyouras** [comment]
Thank you for joining the discussion! 🙏  We'll make sure you have everything you need to develop your plugin.

**Comment by Samuel-Samuel** [comment]
Salut,
Le plugin Jeedom m’intéresse, j'ai ma clé API si tu veux un bêta testeur.

**Reply by mguyard** [reply]
Envoi moi en message sur Community pour que je garde de coté. Pseudo identique

**Reply by Samuel-Samuel** [reply]
c'est fait :)

**Reply by bignooz** [reply]
Ça m'intéresse aussi. Tu veux ma clef ?

**Reply by mguyard** [reply]
J'ai vu avec iopool pour leur faire voir la beta en avant premiere et valider le design. Après cela je pourrais passer en beta et vous pourrez tester avec vos clé API. Mais un peu de patience tout de même..

**Reply by bignooz** [reply]
Tu prévois une historisation des variables?

**Reply by lecomtevalentin** [reply]
Dispo aussi pour test si besoin :)

**Reply by stephanehouse** [reply]
Moi également, très bonne nouvelle cette api 🙂

**Reply by Seth3d85** [reply]
Bonjour, quelqu’un pour m’expliquer comment faire une intégration de iopool sous home assistant ?

**Comment by gtrhayrug** [comment]
Hello, bonne idée ! Je suis le sujet ;).

**Comment by mguyard** [comment]
### iopool Jeedom plugin is now released.
All documentation is publicly available : https://mguyard.github.io/Jeedom-Documentations/fr_FR/iopool_EcO/documentation

What is available in your Jeedom interface ? I think you will be not lost

![widget](https://user-images.githubusercontent.com/5262395/132034279-6db74097-9ef4-4ac6-9de6-91b6f3fdbc7f.png)
_Yes i know this summer isn't really a summer :(_

### And the better it's that's available for **FREE**....

**Comment by mguyard** [comment]
Hello,

The opening of your pool will start in few weeks.
iopool assists us in the management of our pools, even indicating the recommended filtration time.

Today, iopool Jeedom plugin was released in 2.0.1 BETA version to include a new feature, the ability to manage your filtration system if you use a connected plug, based on filtration recommandation time provide by iopool.

![FiltrationSnap](https://user-images.githubusercontent.com/5262395/157923888-87fb1d63-877f-40f0-9f32-ba8ef5da69bd.png)

Documentation is available here : https://mguyard.github.io/Jeedom-Documentations/fr_FR/iopool_EcO/documentation#Filtration
Sorry for the French only documentation (plugin interface is available in English) but it's easy to understand how to configure this feature.

This version also include filtering Boost (also available in Homekit thru Jeedom Homebridge plugin) in addition of daily filtering time to support when you add chemicals

Objective is to switch version from BETA to STABLE in few weeks.
